src/app/saved-jobs/placeHolderData.ts

```diff
@@ -0,0 +1,74 @@
+export type Job = {
+    jobTitle: string;
+    company: string;
+    location: string;
+    savedDate: string;
+    savedLocation: string;
+    companyImage?: string;
+}
+
+export const jobsArr: Array<Job> = [
+    {
+        jobTitle: "React Dev",
+        company: "Some Company",
+        location: "Modesto, CA",
+        savedDate: "Saved Today",
+        savedLocation: "Saved"
+    },
+    {
+        jobTitle: "Web Dev",
+        company: "WebDevelopment Inc",
+        location: "Riverbank, CA",
+        savedDate: "Saved Yesterday",
+        savedLocation: "Saved"
+    },
+    {
+        jobTitle: "NextJS Web Designed",
+        company: "NextJS Studio",
+        location: "Modesto, CA",
+        savedDate: "Saved Yesterday",
+        savedLocation: "Applied"
+    },
+    {
+        jobTitle: "React Props Designed",
+        company: "React Studio",
+        location: "Modesto, CA",
+        savedDate: "Saved Yesterday",
+        savedLocation: "Applied"
+    },
+    {
+        jobTitle: "React State Handler",
+        company: "React Studio",
+        location: "Modesto, CA",
+        savedDate: "Saved Yesterday",
+        savedLocation: "Applied"
+    },
+    {
+        jobTitle: "React Effects Designed",
+        company: "React Studio",
+        location: "Modesto, CA",
+        savedDate: "Saved Yesterday",
+        savedLocation: "Applied"
+    },
+    {
+        jobTitle: "TypeScript Developer",
+        company: "TypeScript Studio",
+        location: "Modesto, CA",
+        savedDate: "Saved Yesterday",
+        savedLocation: "Applied"
+    },
+    {
+        jobTitle: "HTML Web Developer",
+        company: "HTML Development Studios",
+        location: "Riverbank, CA",
+        savedDate: "Saved Last Week",
+        savedLocation: "Archived"
+    }
+]
+
+export const jobsAmounts = {
+    savedJobs: jobsArr.filter(job => job.savedLocation == "Saved").length,
```

Comment:
I know this is just test data (which you implemented nicely) but we should always make it a practice to use `===` instead of `==`